packages/shared_preferences/shared_preferences_linux/example/lib/main.dart

```diff
@@ -31,7 +32,7 @@ class SharedPreferencesDemo extends StatefulWidget {
 }
 
 class SharedPreferencesDemoState extends State<SharedPreferencesDemo> {
-  final prefs = SharedPreferencesLinux.instance;
+  final prefs = SharedPreferencesStorePlatform.instance;
```

Comment:
Same.

Reply:
This could be beneficial in this case as it's also testing that `registerWith` is automatically called by the engine.

Reply:
Ah, good point.